README.md

```diff
@@ -6,58 +6,44 @@
-  versatile stack adhering to the key design principles of modularity and
-  pluggability that define the libp2p project. It adds features present in
-  mature implementations like go-libp2p, rust-libp2p, js-libp2p.
-    * will offer: pluggable peerstore, connection manager, QUIC transport,
-      circuit relay, AutoNAT, AutoRelay, NAT traversal, etc.
-    * Android-friendly.
-    * Kotlin coroutine-based façade, possibly a Reactive Streams façade too.
-    * work will begin after the minimal phase concludes.
-
-## minimal phase (v0.x): Definition of Done
-
-We have identified the following components on the path to attaining a minimal
-implementation:
-
-- [X] multistream-select 1.0
-- [X] multiformats: [multiaddr](https://github.com/multiformats/multiaddr)
-- [X] crypto (RSA, ed25519, secp256k1)
-- [X] [secio](https://github.com/libp2p/specs/pull/106)
-- [X] [connection bootstrapping](https://github.com/libp2p/specs/pull/168)
-- [X] mplex as a multiplexer
-- [X] stream multiplexing
-- [X] TCP transport (dialing and listening)
-- [X] Identify protocol
-- [X] Ping protocol
-- [X] [peer ID](https://github.com/libp2p/specs/pull/100)
-- [X] noise security protocol
-- [X] MDNS
-- [X] Gossip 1.1 pubsub 
-
-We are explicitly leaving out the peerstore, DHT, pubsub, connection manager,
-etc. and other subsystems or concepts that are internal to implementations and
-do not impact the ability to hold communications with other libp2p processes.
+[Libp2p](https://libp2p.io/) implementation for the JVM, written in Kotlin 🔥
+
+## Components
+
+List of components in the Libp2p spec and their JVM implementation status 
+
+|                          | Component                                                                                       |          Status          |
+|--------------------------|-------------------------------------------------------------------------------------------------|:------------------------:|
+| **Transport**            | tcp                                                                                             |    :green_apple:    |
+|                          | [quic](https://github.com/libp2p/specs/tree/master/quic)                                        | :tomato: |
+|                          | websocket                                                                                       |    :lemon:     |
+|                          | [webtransport](https://github.com/libp2p/specs/tree/master/webtransport)                        |                       |
+|                          | [webrtc-browser-to-server](https://github.com/libp2p/specs/blob/master/webrtc/webrtc-direct.md) |                       |
+|                          | [webrtc-private-to-private](https://github.com/libp2p/specs/blob/master/webrtc/webrtc.md)       |                       |
+| **Secure Communication** | [noise](https://github.com/libp2p/specs/blob/master/noise/)                                     |    :green_apple:    |
```

Comment:
Maybe can add secio in secure communication section? https://github.com/libp2p/specs/blob/master/secio/README.md

Reply:
I was deprecated ans is not more in the official list in Libp2p spec

Reply:
However we may add it with 'deprecated' mark :)

Reply:
Ah got it, can ignore it then if deprecated anyways.